src/Air.zig

```diff
@@ -1134,9 +1142,31 @@ pub const CondBr = struct {
 /// Trailing:
 /// * 0. `Case` for each `cases_len`
 /// * 1. the else body, according to `else_body_len`.
+/// * 2. dispatch information.
 pub const SwitchBr = struct {
     cases_len: u32,
     else_body_len: u32,
+    /// 0b000000000000000000000000000000XX - dispatch_mode
+    /// 0bXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX00 - cardinal
+    flags: u32,
```

Comment:
"cardinal" is the number of values the `switch`'s operand type can represent and, consequently, the size of the dispatch table, since all of them must be handled.

Reply:
In that case, just rename it to `dispatch_count` or `dispatch_table_len` or something? No need to use a bespoke term here.